src/ducks/selection/useSelectionBarActions.js

```diff
@@ -0,0 +1,13 @@
+import GraphCircleIcon from 'cozy-ui/transpiled/react/Icons/GraphCircle'
+
+const useSelectionBarActions = showTransactionCategoryModal => {
```

Comment:
mmh, it is named after a hook but it is not a hook. can you rename it with make ? This way it is visible from the caller point of view that the result can/needs to be memoized.

Reply:
nit: The name of the function, apart from looking like the name of the hook is very generic, while the option is very specific, this is a bit disturbing. I am not sure that this function should have been extracted so early.

Reply:
Exact bien vu. J'ai repris l'exemple de Drive, le fait de l'extraire me paraît bien pour centraliser les actions, on souhaitera probablement en rajouter par la suite.